UI/Web/src/app/book-reader/_components/book-reader/book-reader.component.ts

```diff
@@ -1581,17 +1632,27 @@ export class BookReaderComponent implements OnInit, AfterViewInit, OnDestroy {
     }
 
     if (
-      Math.abs(this.mousePosition.x - event.screenX) <= mouseOffset &&
-      Math.abs(this.mousePosition.y - event.screenY) <= mouseOffset
+      Math.abs(this.mousePosition.x - event.clientX) <= mouseOffset &&
+      Math.abs(this.mousePosition.y - event.clientY) <= mouseOffset
     ) {
       this.actionBarVisible = !this.actionBarVisible;
       this.cdRef.markForCheck();
     }
   }
 
   mouseDown($event: MouseEvent) {
-    this.mousePosition.x = $event.screenX;
-    this.mousePosition.y = $event.screenY;
+    this.mousePosition.x = $event.clientX;
+    this.mousePosition.y = $event.clientY;
+    if (!this.clickToPaginate || !this.isCursorOverPaginators($event)) {
+      return
+    }
+
+    const now = Date.now();
+    // This prevents selecting text when clicking repeatedly to switch pages
+    if (now - this.lastPaginatorClickTime < 500) {
```

Comment:
Put this magic number into a const. So debouncing wouldn't work on this?

Reply:
Hmmm now that you mention it, this entire check is unnecessary. I'll remove it

Reply:
Ok, this _may_ have been required for allowing starting a selection from a pagination area